trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
+  end
+end
```

Comment:
- [ ] @GhostEsso Great work implementing the changes requested however you need to change the class name from TrimmerDecorator to CapitalizeDecorator, I also noted the output from your tests displays Unknown for the username kindly check your person class and ensure it's working correctly.
![image](https://github.com/GhostEsso/School_library/assets/61408860/b9075a7f-87f6-453d-8930-15686dd2fc28)